lib/models/project.js

```diff
@@ -564,11 +564,14 @@ Project.prototype.generateTestFile = function(/* moduleName, tests */) {
   @private
   @static
   @method closest
-  @param  {String} pathName Path to your project
+  @param  {String} pathName Path to your project (deprecated)
```

Comment:
@jeffjewiss sorry for nitpicking, but we should probably use the `@deprecated` annotation here roughly like this:

```js
/**
   Returns a new project based on the first package.json that is found
   in `pathName`.
 
    @deprecated
    @private
    @static
    ...
*/
```